.github/workflows/pr-code-format.yml

```diff
@@ -31,12 +28,13 @@ jobs:
           separator: ","
           skip_initial_fetch: true
 
-      # We need to make sure that we aren't executing/using any code from the
-      # PR for security reasons as we're using pull_request_target. Checkout
-      # the target branch with the necessary files.
+      # We need to pull the script from the main branch, so that we ensure
+      # we get the latest version of this script.
       - name: Fetch code formatting utils
         uses: actions/checkout@v4
         with:
+          reository: ${{ github.repository }}
```

Comment:
reository -> repository.

Reply:
f6c87be1dd24a121d7eccd6b91ca808ecdf80356

Thanks for pointing this out!